modules/microshift-nw-multus-add-pod.adoc

```diff
@@ -0,0 +1,175 @@
+[source,yaml]
+----
+metadata:
+  annotations:
+    k8s.v1.cni.cncf.io/networks: |-
+      [
+        {
+          "name": "<network>", <1>
+          "namespace": "<namespace>", <2>
+          "default-route": ["<default-route>"] <3>
+        }
+      ]
+----
+<1> Specify the name of the additional network defined by a `NetworkAttachmentDefinition` object.
+<2> Specify the namespace where the `NetworkAttachmentDefinition` object is defined.
+<3> Optional: Specify an override for the default route, such as `192.168.17.1`.
+
+. To create a `Pod` YAML file and add the annotation for an additional network, run the following command and use the following example YAML:
++
+[source,terminal]
+----
+$ oc apply -f ./<test-bridge>.yaml <1>
+----
+<1> Replace _<test-bridge>_ with the pod name that you want to use.
++
+.Example output
+----
+pod/test-bridge created
+----
++
+. Make sure that the NetworkAttachmentDefinition annotation is correct:
++
+.Example NetworkAttachmentDefinition annotation
++
+[source,terminal]
+----
+metadata:
+  annotations:
+    k8s.v1.cni.cncf.io/networks: bridge-conf
+----
++
+.Example `test-bridge` pod YAML
+[source,terminal]
+----
+kind: Pod
+apiVersion: v1
+metadata:
+  name: test-bridge
+  annotations:
+    k8s.v1.cni.cncf.io/networks: bridge-conf
+  labels:
+    app: test-bridge
+spec:
+  terminationGracePeriodSeconds: 0
+  containers:
+  - name: hello-microshift
+    image: quay.io/microshift/busybox:1.36
+    command: ["/bin/sh"]
+    args: ["-c", "while true; do echo -ne \"HTTP/1.0 200 OK\r\nContent-Length: 16\r\n\r\nHello MicroShift\" | nc -l -p 8080 ; done"]
+    ports:
+    - containerPort: 8080
+      protocol: TCP
+    securityContext:
+      allowPrivilegeEscalation: false
+      capabilities:
+        drop:
+        - ALL
+      runAsNonRoot: true
+      runAsUser: 1001
+      runAsGroup: 1001
+      seccompProfile:
+        type: RuntimeDefault
+----
+
+. Optional: To confirm that the annotation exists in a `Pod` YAML, run the following command, replacing `<name>` with the name of the pod.
++
+[source,terminal]
+----
+$ oc get pod <name> -o yaml <1>
+----
+<1> Replace _<name>_ with the pod name you want to use. In the following example, `test-bridge` is used.
++
+In the following example, the `test-bridge` is attached to the `net1` additional network:
++
+[source,terminal]
+----
+$ oc get pod test-bridge -o yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  annotations:
+    k8s.v1.cni.cncf.io/networks: bridge-conf
+    k8s.v1.cni.cncf.io/network-status: |- <1>
+      [{
+          "name": "ovn-kubernetes",
+          "interface": "eth0",
+          "ips": [
+              "10.42.0.18"
+          ],
+          "default": true,
+          "dns": {}
+      },{
+          "name": "bridge-conf",
+          "interface": "net1",
+          "ips": [
+              "20.2.2.100"
+          ],
+          "mac": "22:2f:60:a5:f8:00",
+          "dns": {}
+      }]
+  name: pod
+  namespace: default
+spec:
+  ...
+status:
+  ...
```

Comment:
This part is output of `oc get pod test-bridge -o yaml`, right? Should be a new block if so. (also, that allows you to tag it as YAML and get highlighting for it instead of all grey-on-grey as it appears currently)